R/plot-build.r

```diff
@@ -205,7 +205,8 @@ ggplot_gtable.ggplot_built <- function(data) {
     } else {
       # x and y are adjusted using justification of legend box (i.e., theme$legend.justification)
       legend_box <- editGrob(legend_box,
-        vp = viewport(x = xjust, y = yjust, just = c(xjust, yjust)))
+        vp = viewport(x = xjust, y = yjust, just = c(xjust, yjust),
+          height = legend_height, width = legend_width))
```

Comment:
Code formatting seems a bit haphazard here. The same code a few lines further up, which you didn't touch, has the same issue. I suggest fixing both.

In terms of logic, looks good to me.

Reply:
😄 I was just trying to match the weird formatting on top